doc/HOWTOs.org

```diff
@@ -68,3 +69,17 @@ Add the following snippet to your =dostpacemacs/config= function:
     (define-key evil-visual-state-map "j" 'evil-next-visual-line)
     (define-key evil-visual-state-map "k" 'evil-previous-visual-line)
 #+end_src
+
+** Disable evilification of a mode
+You can ensure a mode opens in emacs state by using =evil-set-initial-state=.
+
+#+begin_src emacs-lisp
+  (evil-set-initial-state 'magit-mode 'emacs)
```

Comment:
Well, it was just an example. I can change it to something that *actually* works.